src/structures/Guild.js

```diff
@@ -534,6 +534,30 @@ class Guild extends AnonymousGuild {
     );
   }
 
+  /**
+   * Return maximum bitrate based on the guild premium_tiers level and the features
```

Comment:
Personally, I'd change this line to this:
```suggestion
   * The maximum bitrate available for this guild
```
I think it's more concise? Just throwing this out there!